mailbox/api/src/main/java/org/apache/james/mailbox/DefaultMailboxes.java

```diff
@@ -38,7 +38,7 @@ public interface DefaultMailboxes {
     String TEMPLATES = "Templates";
     String RESTORED_MESSAGES = "Restored-Messages";
 
-    List<String> DEFAULT_MAILBOXES = ImmutableList.of(INBOX, OUTBOX, SENT, TRASH, DRAFTS, SPAM);
+    List<String> DEFAULT_MAILBOXES = ImmutableList.of(INBOX, OUTBOX, SENT, TRASH, DRAFTS, ARCHIVE, SPAM);
```

Comment:
Pretty sure this will cause some tests to fail (JMAP mailbox provisionning).

Those tests would need to be fixed, too.

Reply:
Unfortunately I only run test of the modules I modified, mostly due to all suite with all integration test taking hours (probably https://issues.apache.org/jira/browse/JAMES-3879 would help with "test-ability" here :-) ).

What's the desired way forward - keeping ARCHIVE as default (I would say it's quite elementary folder) and identifying all those test or removing it from the list of defaults?

As for testing goes, I checked the build (https://ci-builds.apache.org/job/james/job/ApacheJames/job/PR-1476/1/testReport/) and there are a lot of failures, but from the cursory look most of them are due to configuration not being set during tests causing nasty NPE so most likely a missing class/injection in the tests...